swift/Arrow/Package.swift

```diff
@@ -0,0 +1,23 @@
+// swift-tools-version:5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
```

Comment:
Can we specify package license?

Reply:
I am not sure what you mean.  I didn't see an attribute on Package for a license.

Reply:
Most package formats can include license information.
Examples:
* Python package:  https://pypi.org/project/pyarrow/ " License OSI Approved :: Apache Software License"
* Ruby package: https://rubygems.org/gems/red-arrow "License: Apache-2.0"

I'm sorry but I'm not familiar with Swift package format. If Swift package format supports license information, we should use it.

Reply:
I see what you mean.  I have looked at some other repos and the Package documentation and it doesn't look like the Package.swift file contains any licensing info besides the one included in the header.  

Once this PR has been merged, I will look at registering this package in https://swiftpackageregistry.com.

Reply:
Gotcha, thanks for letting me know.

Reply:
@abandy I am not familar with swift how is it distributed? Are the binary artifacts that need to be built and distributed or is it more in line with go where a github tag is the only thing required?

Reply:
It would be more inline with go.  Packages are included using git url and tag.